src/garage/envs/dm_control/dm_control_env.py

```diff
@@ -22,22 +24,68 @@ def _flat_shape(observation):
     return np.sum(int(np.prod(v.shape)) for k, v in observation.items())
 
 
-class DmControlEnv(gym.Env):
+class DmControlEnv(Environment):
     """Binding for `dm_control <https://arxiv.org/pdf/1801.00690.pdf>`."""
 
-    def __init__(self, env, name=None):
+    def __init__(self, env, name=None, max_episode_length=math.inf):
```

Comment:
dm_control envs already have a time limit: https://github.com/deepmind/dm_control/blob/fb0199e61db3323a17f56fd089af61f7c949f41e/dm_control/rl/control.py#L41

Reply:
we should use it in the same way we use the time limits from gym envs.